edit/persistentvolume/index.vue

```diff
@@ -44,13 +44,15 @@ export default {
   async fetch() {
     const storageClasses = await this.$store.dispatch('cluster/findAll', { type: STORAGE_CLASS });
     const pvcPromise = this.$store.dispatch('cluster/findAll', { type: PVC });
+    const featureFlagPromise = this.$store.dispatch('management/find', { type: MANAGEMENT.FEATURE, id: 'unsupported-storage-drivers' });

```

Comment:
This needs to be wrapped in a check for  `getters['isRancher']`; Standalone modes are not going to have a feature flag type and will fail.

Reply:
Done. I made a feature-flag util that will account for this if people need to use it in the future.